src/erl/erlang/erlang.io.File.erl

```diff
@@ -79,13 +87,13 @@ read(File) ->
 read(#{?TYPE := ?M, pid := Pid}, Length) ->
   case io:get_chars(Pid, "", Length) of
     eof -> eof;
-    Str -> list_to_binary(Str)
+    Str -> unicode:characters_to_binary(Str)
```

Comment:
This is a fix for a bug triggered when reading using a `PushbackReader`, if there was a special character then it wasn't encoded correctly.